cmake_modules/FindMKL.cmake

```diff
@@ -0,0 +1,144 @@
+# This is from the Marian project, which is under MIT license
+# https://github.com/marian-nmt/marian-dev
+
```

Comment:
If this is MIT License, we need to include the original copyright notice right?

Reply:
Yes you're probably right. I now added the MIT license and copyright notice for that whole project, though to be honest it looks to me like that particular file might have its own separate copyright (different author). Either way, all the information is now included.